htlcswitch/link.go

```diff
@@ -599,19 +608,19 @@ func (l *channelLink) markReestablished() {
 }
 
 // sampleNetworkFee samples the current fee rate on the network to get into the
-// chain in a timely manner. The returned value is expressed in fee-per-kw, as
-// this is the native rate used when computing the fee for commitment
-// transactions, and the second-level HTLC transactions.
-func (l *channelLink) sampleNetworkFee() (chainfee.SatPerKWeight, error) {
-	// We'll first query for the sat/kw recommended to be confirmed within 3
-	// blocks.
-	feePerKw, err := l.cfg.FeeEstimator.EstimateFeePerKW(3)
```

Comment:
Since anchors are the default channel type in lnd, we're effectively dropping our fee rates for all our commitments to conf target = 3 to 6 with this change. Are we good with that?

Reply:
yeah good question. Perhaps we should leave it at 3 for now and let those who want to change it change it?
Just seems weird to use anchor channels and then still set a high conf target. But maybe it is best to leave it as is while we wait for package relay? thoughts @Roasbeef ?

Reply:
yeah i'm happy to change it back. The thinking was just that it is kinda weird to have anchor commitment fees be so high since it kinda defeats the purpose since, no? or do we ideally _not_ want to have to bump the fee? thoughts appreciated @Roasbeef  & @carlaKC